features/changed.feature

```diff
@@ -134,4 +134,32 @@ Feature:  changed
       | OFF           | OFF   | should not |
       | OFF           | ON    | should     |
       | ON            | ON    | should not |
-      | ON            | OFF   | should     |
\ No newline at end of file
+      | ON            | OFF   | should     |
+
+  Scenario: GroupMembers are separated from items in triggers
+    Given group "Switches"
+    And group "Contacts"
+    And items:
+      | type    | name     | group    |
+      | Switch  | Switch1  | Switches |
+      | Switch  | Switch2  | Switches |
+      | Switch  | Switch3  |          |
+      | Contact | Contact1 | Contacts |
+      | Contact | Contact2 | Contacts |
+    And a deployed rule:
+      """
+      rule 'Nested groups' do
+        changed Switches.members, [[Contacts.members], Switch3]
+        triggered { |item| logger.info("#{item.id} triggered the rule") }
+      end
+      """
+    When I add items:
+      | type    | name     | group    |
+      | Contact | Contact3 | Contacts |
```

Comment:
You can set the item state here too.

Reply:
Should I remove that from the Examples section below then, and just have the state set there? Then I would have to add a separate 'It should log ...' statement for that item?

Reply:
This is fine, I misunderstood what was happening here.